app/models/directory_item.rb

```diff
@@ -148,6 +110,133 @@ def self.refresh_period!(period_type, force: false)
     end
   end
 
+  def self.prepare_refresh_period_cte
+    <<~SQL
+      WITH humans AS (
```

Comment:
```suggestion
      WITH eligible_users AS (
```

Minor nit since this cte filters for more than just human users.

Reply:
Yep you are right, will change